foundations/installations/installation_overview.md

```diff
@@ -47,6 +51,8 @@ A **virtual machine** is an emulation of a computer that runs within your existi
 
 Dual-booting can be as easy as inserting a flash drive and clicking a few buttons. The benefits of dual-booting cannot be overstated. You will allow Linux to access the full capabilities of your hardware, have a clean and distraction-free environment for coding, and learn the platform used by many senior developers and servers around the world.
 
+**Windows Subsystem for Linux**, or WSL2 as we'll refer to it from now on, is a feature that lets users run a Linux environment directly inside Windows without the need to run a virtual machine. TODO: Insert further copy here.
```

Comment:
note on this below as well. Stay consistent with naming it either `WSL` or `WSL2`